aslprep/config.py

```diff
@@ -384,7 +384,7 @@ def init(cls):
 
         if cls._layout is None:
             import re
-            from bids.layout import BIDSLayout
+            from aslprep.pybids.layout import BIDSLayout
```

Comment:
I replaced any general imports of the four external libraries with internal imports, so that I know for sure where everything's coming from. I'm hoping it'll make it easier to systematically remove the external libaries down the line.

I don't know what changes were made to pybids. Hopefully none, so it would be easy to remove.